placeholder_accessors/index.md

```diff
@@ -192,6 +193,51 @@ The handler gains a new method,
 that registers the requirements of the placeholder accessor on the given
 command group.
 
+### Placeholder `accessor` without a buffer
+
+Not providing either the handler or the memory object
+makes the accessor default constructible,
+creating a "null" accessor.
```

Comment:
a "null" accessor -> a "null" or "unbound" placeholder accessor. This represents just the placeholder (i.e. the space that will be occupied by the data type) that needs to be associated with a buffer at command group submission time.